cmd/dependabot/internal/cmd/test.go

```diff
@@ -37,22 +37,25 @@ var testCmd = &cobra.Command{
 		processInput(&scenario.Input)
 
 		if err := infra.Run(infra.RunParams{
-			CacheDir:      cache,
-			Creds:         scenario.Input.Credentials,
-			Debug:         debugging,
-			Expected:      scenario.Output,
-			ExtraHosts:    extraHosts,
-			InputName:     file,
-			InputRaw:      inputRaw,
-			Job:           &scenario.Input.Job,
-			LocalDir:      local,
-			Output:        output,
-			ProxyCertPath: proxyCertPath,
-			ProxyImage:    proxyImage,
-			PullImages:    pullImages,
-			Timeout:       timeout,
-			UpdaterImage:  updaterImage,
-			Volumes:       volumes,
+			CacheDir:            cache,
+			Creds:               scenario.Input.Credentials,
+			Debug:               debugging,
+			EnableOpenTelemetry: enableOpenTelemetry,
+			Expected:            scenario.Output,
+			ExtraHosts:          extraHosts,
+			InputName:           file,
+			InputRaw:            inputRaw,
+			Job:                 &scenario.Input.Job,
+			LocalDir:            local,
+			Output:              output,
+			ProxyCertPath:       proxyCertPath,
+			ProxyImage:          proxyImage,
+			CollectorConfigPath: collectorConfigPath,
```

Comment:
nit: looks like the rest of these parameters are ordered alphabetically.

Reply:
Thanks! I didn't realise.